### PR TITLE
Product screen development with data

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull Request I have developed the view of the products page to be listed on Eleganz Store.
List of files to be studied within src folder:

1) context/ProductContext.jsx
2) ProductList and Filter Sidebar components in components folder
3) ProductDisplay folder within `pages` folder

Live preview: https://deploy-preview-6--eleganz-store.netlify.app/products

### CODE REVIEW

File: src/components/ProductCard/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const ProductCard = ({productDetails}) => {
+
+    
+    // eslint-disable-next-line no-unused-vars
+    const {_id,title,categoryName,imgUrl,ratings,originalPrice,discountedPrice,discount,brand,inStock,wornBy}=productDetails
+    return (      
+        <div key={_id} className={`card card-vertical ${!inStock?"card-overlay":""}`}>
+            {!inStock?<div className="overlay-text-container">
+                <div className="card-overlay-text">OUT OF STOCK</div>
+            </div>:null}
```
Comment (JantuDeb) [inline src/components/ProductCard/ProductCard.jsx:14]:
```
!inStock && <div className="overlay-text-container">
                          <div className="card-overlay-text">OUT OF STOCK</div>
                    </div>
```

**Review by JantuDeb — APPROVED** [review]
left a comment

File: src/components/ProductCard/ProductCard.jsx (hunk 1 above)
Comment (dev-enforced) [reply src/components/ProductCard/ProductCard.jsx:14]: Had a confusion between when and where to use ternary ones and the conditional AND operator  &&?

**Review by harshalini — APPROVED** [review]
The UI is pretty cool, I also liked the name of your ecommerce. File structure is nice,  I liked that you have made the page components separately in small chunks!

File: src/components/ProductCard/ProductCard.jsx (hunk 1 above)
Comment (dev-enforced) [reply src/components/ProductCard/ProductCard.jsx:14]: Got the understanding about it!!Thank you

**Pull request merged**